python/ray/tune/trainable.py

```diff
@@ -330,6 +330,15 @@ def restore(self, checkpoint_path):
         self._timesteps_since_restore = 0
         self._iterations_since_restore = 0
         self._restored = True
+        logger.info("Restored from checkpoint: {}".format(checkpoint_path))
+        state = {
```

Comment:
Let's change these keys to match the attribute names:
```python
state = {
    "_iteration": self._iteration,
    "_timesteps_total": self._timesteps_total,
    "_time_total": self._time_total,
    "_episodes_tota": self._episodes_total
}
```

It's easier to interpret and debug when the logging names match the attribute names exactly.